src/main/java/uk/gov/hmcts/reform/sendletter/services/ftp/FtpClient.java

```diff
@@ -131,6 +131,36 @@ public void deleteReport(String reportPath) {
         });
     }
 
+    public List<FileInfo> listLetters(String serviceFolder) {
+        return runWith(sftp -> {
+            try {
+                String path = String.join("/", configProperties.getTargetFolder(), serviceFolder);
+                return sftp
+                    .ls(path)
+                    .stream()
+                    .map(it -> new FileInfo(
+                            it.getPath(),
+                            Instant.ofEpochSecond(it.getAttributes().getMtime())
+                        )
```

Comment:
Perhaps CI will pick this up - looks like incorrect indentation. I might be wrong :)

Reply:
checkstyle / intellij think it's fine.